### ISSUE TITLE
UX/accessibility Issue - Cannot reach the accounts buttons switches with tab on Linked Account

### ISSUE THREAD

**Issue by PereViki** [topcoder-platform/community-app#1403]
**Describe the bug**
All affordances should be available through tabbing through the site, however, account buttons (Github, Behance etc.) buttons cannot be reached from the keyboard.

**To Reproduce**
1. Go to 'https://www.topcoder-dev.com/settings/account'
2. Click on 'Linked Account'
3. Tab through page

**Expected behavior**
The user should be able to link an account from the keyboard.

**Screenshots**
http://www.screencast.com/t/ybgtasULS

**Desktop: MacBook Pro**
 - OS: Mac OS High Sierra
 - Browser: Chrome
 - Version: 69.0.3497.92

@ThomasKranitsas
